share/schema_versions/App-Netdisco-DB-53-54-PostgreSQL.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+ALTER TABLE device_port ADD COLUMN "ifindex" bigint;
```

Comment:
I think this is better off in the device_port_properties table, which is storing the 'raw' data for other fields too.
```suggestion
ALTER TABLE device_port_properties ADD COLUMN "ifindex" bigint;
```

Reply:
Alright, I wasn't quite aware what should go into _port and _port_properties respectively. I'll update the PR